src/coreclr/src/vm/gccover.cpp

```diff
@@ -676,31 +613,28 @@ void GCCoverageInfo::SprinkleBreakpoints(
             break;
         }
 
-        if (!skipGCForCurrentInstr)
+        if (prevDirectCallTargetMD != 0)
         {
-            if (prevDirectCallTargetMD != 0)
-            {
-                ReplaceInstrAfterCall(cur, prevDirectCallTargetMD);
-            }
+            ReplaceInstrAfterCall(cur, prevDirectCallTargetMD);
+        }
 
-            // For fully interruptible code, we end up whacking every instruction
-            // to INTERRUPT_INSTR.  For non-fully interruptible code, we end
-            // up only touching the call instructions (specially so that we
-            // can really do the GC on the instruction just after the call).
-            size_t dwRelOffset = (cur - codeStart) + regionOffsetAdj;
-            _ASSERTE(FitsIn<DWORD>(dwRelOffset));
-            if (codeMan->IsGcSafe(&codeInfo, static_cast<DWORD>(dwRelOffset)))
-                *cur = INTERRUPT_INSTR;
+        // For fully interruptible code, we end up whacking every instruction
+        // to INTERRUPT_INSTR.  For non-fully interruptible code, we end
+        // up only touching the call instructions (specially so that we
+        // can really do the GC on the instruction just after the call).
+        size_t dwRelOffset = (cur - codeStart) + regionOffsetAdj;
+        _ASSERTE(FitsIn<DWORD>(dwRelOffset));
+        if (codeMan->IsGcSafe(&codeInfo, static_cast<DWORD>(dwRelOffset)))
+        *cur = INTERRUPT_INSTR;
```

Comment:
Indentation is off here.

Reply:
Think its ok? I removed a level of nesting here since we no longer check `if (!skipGCForCurrentInstr)`.

Reply:
Shouldn't the `*cur = INTERRUPT_INSTR;` be indented?

Reply:
Indeed. Will fix.